spec/rails_helper.rb

```diff
@@ -60,8 +60,9 @@
   # Add modules for helpers
   config.include ControllerSpecHelper, type: :controller
   config.include RequestSpecHelper, type: :request
-  [:feature, :request].each do |type|
+  [:feature, :request, :model].each do |type|
     config.include IntegrationSpecHelper, type: type
+    config.include EnvironmentSpecHelper, type: type
```

Comment:
We don't want to include the `IntegrationSpecHelper` for models...mind moving this line out of the loop to be `config.include EnvironmentSpecHelper, type: :model`?